layouts/shortcodes/activities-matrix.html

```diff
@@ -0,0 +1,11 @@
+{{ $matrix_activity := partial "activities-matrix.html" . }}
+
+{{ if eq .Site.Language.Lang "de" }}
+    Im <a href="https://matrix.to/#/#berlin.freifunk.net:matrix.org">Freifunk Berlin Matrix Space</a> gibt es aktuell
+    <b>{{ $matrix_activity.room_count }} Chat-Räume mit {{ $matrix_activity.member_count }} Usern</b>.
+    Im Hauptchat wurden in den letzten sieben Tagen <b>{{ $matrix_activity.message_count }} Nachrichten</b> verschickt.
```

Comment:
Ich würde den Zeitraum auf 30 Tage erhöhen, da es immer wieder Phasen gibt, wo im Chat nicht viel los ist.

Reply:
Resolved. Current live data might not be correct, because the account used for collecting this data is no 30 days old.